scripts/runWorker.pl

```diff
@@ -67,8 +67,8 @@ sub main {
                'worker_cur_dir|cwd=s'       => \$worker_cur_dir,
                'hive_log_dir|hive_output_dir=s'         => \$hive_log_dir,       # keep compatibility with the old name
                'worker_log_dir|worker_output_dir=s'     => \$worker_log_dir,     # will take precedence over hive_log_dir if set
-               'retry_throwing_jobs=i'      => \$retry_throwing_jobs,
-               'can_respecialize=i'         => \$can_respecialize,
+               'retry_throwing_jobs!'       => \$retry_throwing_jobs,
```

Comment:
You'll have to change `-retry_throwing_jobs 1` in docs/running_pipelines/running.rst